backend/MoleculeOSSite/Entities/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+namespace MoleculeOSSite.Entities
+{
+    public class User
+    {
+        public int Id { get; set; }
+        public string Username { get; set; }
+        public string Email { get; set; }
+        public DateTime JoinDate { get; set; }
+        public string PasswordHash { get; set; }
+
+        public int RoleId { get; set; } = 1; //Each user gets "User" role by default
```

Comment:
```suggestion
namespace MoleculeOSSite.Entities
{
    public class User
    {
        public int Id { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public DateTime JoinDate { get; set; }
        public string PasswordHash { get; set; }

        public int RoleId { get; set; } = 1; // Each user gets "User" role by default
```
1. Pusta linia na górze
2. W komentarzach lepiej dawać spacje pomiędzy "//" a treścią komentarza